src/app/asset-page/asset-page.component.pug

```diff
@@ -74,16 +74,16 @@
 
       .col-md-12.quiz-caption-cntnr(*ngIf="isFullscreen", [ngClass]="[ showAssetCaption ? '' : 'cntnr-bg' ]")
         ang-promo-tooltip(*ngIf="user.isLoggedIn && !quizModeTTDismissed && studyMode", [options]="quizModeTooltipOpts", (close)="closeQuizModeTooltip()")
-        .caption-btn-cntnr([ngClass]="[ showAssetCaption ? 'active' : '' ]", tabindex="0", (click)="showAssetCaption = !showAssetCaption", (keydown.enter)="showAssetCaption = !showAssetCaption", title="{{ showAssetCaption ? 'Hide Asset Meta Information' : 'Show Asset Meta Information' }}")
+        .caption-btn-cntnr([ngClass]="[ showAssetCaption ? 'active' : '' ]", tabindex="6", (click)="showAssetCaption = !showAssetCaption", (keydown.enter)="showAssetCaption = !showAssetCaption", title="{{ showAssetCaption ? 'Hide Asset Meta Information' : 'Show Asset Meta Information' }}")
```

Comment:
Forgive me if this has already been discussed on other pull requests but I wonder if we could fix the tab ordering without using the tabindex. It seems that it might be rather tricky to maintain. Moving around the ordering of items in the dom without any tab indexing would probably be the most clear to me but it may not be possible/feasible.

Reply:
Oh yes, that would be perfect if we can do that. But the problem is, we have two components: `artstor-viewer` and `asset-page`. `artstor-viewer` is a child component of `asset-page`. The viewer, thumbnails in the reference strip, zoom buttons, and pagination buttons are in `arstor-viewer`, while the compare and exit button, and quiz buttons are in `asset-page` component for implementation reasons. If I give `tabindex=0` to all of them, the elements inside `arstor-viewer` will always get focus first (not the order we want).